lib/ldclient-rb/ldclient.rb

```diff
@@ -182,6 +182,31 @@ def track(event_name, user, data)
       add_event({:kind => 'custom', :key => event_name, :user => user, :data => data })
     end
 
+    #
+    # Returns the key of every feature
+    #
+    def feature_keys
+      get_features.map {|feature| feature[:key]}
+    end
+
+    #
+    # Returns all features
+    #
+    def get_features
+      res = @client.get (@config.base_uri + '/api/features') do |req|
```

Comment:
One minor thing to consider here is that this will give you the version of the feature as it is seen on the dashboard, which may not be "live" yet due to the TTL:

http://apidocs.launchdarkly.com/v1.0/docs/flag-evaluation-overview

I think we should address this on our end by adding an `/api/eval/features` resource to our API, and then pointing this to that new resource.

I'll take care of that and change the call here. Once I've done that this looks good to go. Thanks for the contribution!